app/code/Magento/Catalog/Ui/DataProvider/CatalogEavValidationRules.php

```diff
@@ -33,14 +33,16 @@ public function build(ProductAttributeInterface $attribute, array $data)
             $rules['validate-zero-or-greater'] = true;
         }
 
-        $validationClasses = explode(' ', $attribute->getFrontendClass());
+        $validationClasses = explode(' ', $attribute->getFrontendClass() ?? '');
```

Comment:
Hello @SilinMykola 
What do you think about suggestion below

```suggestion
        $validationClasses = $attribute->getFrontendClass() ? explode(' ', $attribute->getFrontendClass()) : [];
```

Reply:
Hello @andrewbess 
Thanks for your review. I made changes, please check.